hyp3_testing/compare.py

```diff
@@ -7,8 +7,7 @@
 from os import listdir
 from pathlib import Path
 
-# cv2 is a .so file, so mypy can't find it
```

Comment:
What's the context behind this change? Is mypy no longer showing an `import-not-found` error for the `import cv2` line?